main/main.c

```diff
@@ -142,10 +196,29 @@ run_signal_handler(void)
 int
 main(int argc, char **argv)
 {
+	struct args args = {
+		.lua_base_dir = NULL,
```

Comment:
1. Replace `NULL` with `LUA_BASE_DIR` since this is a default value.

2. Add comment `/* Defaults. */`  just before the line above.